nncf/torch/quantization/layers.py

```diff
@@ -841,6 +850,8 @@ class AsymmetricQuantizer(BaseQuantizer):
     INPUT_RANGE_PARAM_NAME = "input_range"
     _INPUT_RANGE_PARAM_STORAGE_ATTR = "_input_range_param_storage"
 
+    input_low: CompressionParameter
```

Comment:
Could you explain the motivation for this change? Why does the current implementation is wrong?

Reply:
After remove torch2 backend it no need, removed